test/experiment/ab_test.rb

```diff
@@ -37,15 +37,18 @@ def test_requires_at_least_two_alternatives_per_experiment
     assert_raises RuntimeError do
       new_ab_test :none do
         alternatives []
+        default nil
```

Comment:
Are these new `defaults` necessary? Won't it use the first alternative if none is specified?